_posts/2022-09-15

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: Disposables and Dependency Injection
```

Comment:
```suggestion
title: Dispsing once, disposing twice, gone!
```

thoughts about this title? I was thinking about "auction style" announcement but not sure it's recognized easily?

Reply:
I like it!